server.js

```diff
@@ -1,21 +1,31 @@
 'use strict';
 
-var express = require('express'),
+var express = require('express')
   // NOTE: require in our request proxy module
-
-  port = process.env.PORT || 3000,
-  app = express();
-
+var requestProxy = require('express-request-proxy')
+var app = express();
+var port = process.env.PORT || 3000
 // NOTE: now use our proxy within a function to request
 //        our github data on the server.
 
 
 app.use(express.static('./'));
 
-app.get('*', function(request, response) {
-  console.log('New request:', request.url);
-  response.sendFile('index.html', { root: '.' });
-});
+var proxyGHRequest = ('/github/*', function(request, response) {
+  console.log(`Routing request for ${request.params[0]}`);
+  (requestProxy({
+    url: `https://api.github.com/${request.params[0]}`,
+    headers: {
+      Authorization: `token ${process.env.GITHUB_TOKEN}`
+    }
+  }))(request, response)
+})
+
+app.get('/github/*', proxyGHRequest);
```

Comment:
This looks perfect! Nice work setting up your server route to make a request to GitHub for your client!